doc/api/http2.md

```diff
@@ -740,6 +739,21 @@ argument identifying the frame type, and an integer argument identifying the
 error code. The `Http2Stream` instance will be destroyed immediately after the
 `'frameError'` event is emitted.
 
+#### Event: `slowRead`
```

Comment:
The name is misleading, it should really be "slow write".

Reply:
I think `slowRead` makes sense, especially since that seems to be the commonly accepted name. Perhaps it should be `slowReadTimeout` instead though to make it obvious what it is?